Jenkinsfile

```diff
@@ -199,8 +199,11 @@ spec:
                                      file(credentialsId: "${env.SQ_KEY_STORE}", variable: 'KEYSTORE')]) {
                         configFileProvider([configFile(fileId: "${env.CONFIG_FILE}", variable: 'MAVEN_SETTINGS')]) {
                             withSonarQubeEnv(installationName: 'sonar') {
-                                sh 'mvn -s $MAVEN_SETTINGS org.sonarsource.scanner.maven:sonar-maven-plugin:3.7.0.1746:sonar -Dsonar.projectKey="Palisade-Services-${BRANCH_NAME}" -Dsonar.projectName="Palisade-Services-${BRANCH_NAME}" -Dsonar.webhooks.project=$SONARQUBE_WEBHOOK -Djavax.net.ssl.trustStore=$KEYSTORE -Djavax.net.ssl.trustStorePassword=$KEYSTORE_PASS'
-                            }
+                                if (env.CHANGE_BRANCH) {
```

Comment:
Ths looks like another bad merge though

Reply:
@dev930018 this will be removed as its part of a different story, it was just implemeted for sq code smell transfer between pal and PR. But note I tried using the GIT_BRANCH_NAME but for some reason it couldnt expand the variable and the output was "Palisade-services- ". hence this non favourable if statement.
The ticket in the sprint for the correct implemetation of this solution will hopefully find a better way. 

Id suggest leaving this in until its about to be merged so we can track the code smells